docs/network/wwivnet.md

```diff
@@ -72,12 +74,15 @@ Anywhere you see a command to run (wwivconfig, network3, etc), remember this ass
 │ Directory : nets\wwivnet\                                      │
 └────────────────────────────────────────────────────────────────┘
 ```
+
 * Create \wwiv\nets\wwivnet\callout.net with ```@1 && /60 "password"``` 
   replacing password with the one provided by 1@1. *NOTE* The double quotes
   are necessary in the file.
 * Run ```network3 y``` which will verify your configuration and files and
   send an email to your local #1 mailbox. If no network number is given, .0
   is assumed, so ```network3 y``` is the same as ```network3 y .0``` 
+* From the \wwiv run ```bbs.exe -M``` to pickup and deliver the mail.  
+**Note:** The default config of WWIV5TelentServer runs the local node with the -M command so if you run the local node to check your mail this will happen anyway, but you have to launch the local node AFTER you run Network3.
```

Comment:
Did you mean wwivd here? There is no wwiv5telnet server anymore.

Reply:
Ah, yeah, that's from the old docs.  Fixing that...